src/dal-plugin/PlugIn.mm

```diff
@@ -175,6 +175,11 @@ - (void)setPropertyDataWithAddress:(CMIOObjectPropertyAddress)address qualifierD
 - (void)receivedFrameWithSize:(NSSize)size timestamp:(uint64_t)timestamp frameData:(nonnull NSData *)frameData {
     dispatch_sync(_stateQueue, ^{
         if (_state == PlugInStateWaitingForServer) {
+            NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+            [defaults setInteger:size.width forKey:@"obs-virtualcam-dal-width"];
+            [defaults setInteger:size.height forKey:@"obs-virtualcam-dal-height"];
+            [defaults synchronize];
```

Comment:
Actually it seems that it doesn't write synchronously, but it can't hurt.
<img width="737" alt="Bildschirmfoto 2020-05-28 um 10 28 18" src="https://user-images.githubusercontent.com/59806498/83118081-2dc7d700-a0ce-11ea-9f40-056f64bcef98.png">
I was able to remove the [synchronize](https://developer.apple.com/documentation/foundation/nsuserdefaults/1414005-synchronize?language=objc) call though as it is deprecated and `unnecessary and shouldn't be used`.

Reply:
Ah yes, without `synchronize` it'd be fine to do this without the dispatch_async